pkg/server/plugin/datastore/sql/sql_test.go

```diff
@@ -119,6 +121,18 @@ func (s *PluginSuite) TestBundleCRUD() {
 	s.Require().NotNil(fresp)
 	s.Require().Nil(fresp.Bundle)
 
+	// update non-existant
+	_, err = s.ds.UpdateBundle(ctx, &datastore.UpdateBundleRequest{Bundle: bundle})
+	s.Require().Error(err)
```

Comment:
nit: `spiretest.RequireGRPCStatus` might shrink some of this assertion code (or `s.RequireGRPCStatus()` considering PluginSuite is a `spiretest.Suite`).
```
_, err = s.ds.UpdateBundle(ctx, &datastore.UpdateBundleRequest{Bundle: bundle})
s.RequireGRPCStatus(err, codes.NotFound, "bundle not found")
```

Reply:
oh hey that already exists. I'm not used to the Suite style of things :)